### PR TITLE
Fix typo in CONTRIBUTING.md

### LINKED ISSUE #89: Fix typo in CONTRIBUTING.md file
**Describe the bug**
There is a typo in the CONTRIBUTING.md file that breaks the code that hides the URLs list.

**To Reproduce**
* Open the file. 
* Scroll to the end.
* See the mistake.

**Expected behavior**
The URLs list should not show.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by racbc**
Fixes #89.

### Motivation
This PR solves the bug specified in #89.

### Solution
I fixed the typo (switched a { for a [).

### CODE REVIEW

**Review by Emrekur — COMMENTED** [review]
IF0.1
ELSE1.0

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ All support requests must be made via [our support team][3].
 [4]: https://github.com/coinbase/rosetta-sdk-go/pull/294/files
 [5]: https://github.com/coinbase/rosetta-specifications/issues/new/choose
 [6]: https://github.com/coinbase/rosetta-specifications/issues/new?assignees=&labels=bug&template=bug_report.md&title=
-{7]: https://github.com/coinbase/rosetta-specifications/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=
```
Comment (Emrekur) [inline CONTRIBUTING.md:70]: Easy

**Review by Emrekur — APPROVED** [review]
Like

**Comment by cb-heimdall** [comment]
Review Error for Emrekur @ 2022-01-04 15:31:07 UTC
User must have write permissions to review

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**